l10n/exceptions/mozorg.json

```diff
@@ -49,7 +49,9 @@
           "en/footer.ftl:footer-follow-firefox",
           "en/footer.ftl:footer-follow-mozilla",
           "en/mozorg/newsletters.ftl:newsletters-beta-news"
-      ]
+      ],
+      "zh-TW": [
+          "en/mozorg/about/history.ftl:history-the-history-of-firefox-and"
     },
```

Comment:
This is broken. Notice that it's displayed in red? There should be a closing `]` before this line.

Now I need to figure out why the linter didn't reject this.